src/e2e/java/teammates/e2e/cases/InstructorAuditLogsPageE2ETest.java

```diff
@@ -80,6 +80,7 @@ public void testAll() {
 
         studentSubmissionPage.fillTextResponse(1, receiver.getName(), response);
         studentSubmissionPage.clickSubmitQuestionButton(1);
+        syncFeedbackSessionLogEntries();
```

Comment:
I really don't think this is needed. When run in production server, this will cause duplicated logs.

Reply:
Im not sure what is better here? we need to check that the logs are correctly recorded, and in this case manually triggering the cron job seems needed to me

Reply:
@wkurniawan07 also I dont see why this would cause duplicated logs?

Reply:
Let's take a timespan from 10 - 11am. Logs collected by cron jobs are at 10.00, 10.15, 10.30, 10.45 (give/take 1-2 mins).
Run this E2E test at 10.20. Which logs is this test entering to the DB?

Reply:
ok I see, but how do we verify the presence of the created logs? Do we make a call to gcloud logging service directly?